mix.exs

```diff
@@ -4,7 +4,7 @@ defmodule Koans.Mixfile do
   def project do
     [app: :elixir_koans,
      version: "0.0.1",
-     elixir: ">= 1.2.1 and <= 1.4.0",
+     elixir: ">= 1.2.1 and <= 1.4.1",
```

Comment:
@iamvery `~> 1.4` basically means `1.4.x`. 

For `1.5` we would have to update the config still. I think what you're suggesting is something like `>= 1.2.1 and < 2.0.0`. If Elixir follows semver properly, there shouldn't be any breaking changes until 2.0 so I'd be fine with that as well.